src/components/RequestForHelp/index.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { Form, Input, Row, Col, Button } from 'antd';
+import {RegionSelect} from '@components/VolunteerForm/Fields/Select';
+const { TextArea } = Input;
+
+const RequestForHelpForm = props => {
+    
+    const formItemLayout = {
+        labelCol: {
+          xs: { span: 4 },
+          sm: { span: 4 }
+        },
+        wrapperCol: {
+          xs: { span: 24 },
+          sm: { span: 16 }
+        }
+      };
+
+    const buttonLayout = {
+        wrapperCol: {
+            xs: {
+            span: 2,
+            offset: 0
+            },
+            sm: {
+            span: 4,
+            offset: 2
+            }
+        }
+    };
+
+    return (
+        <div style={{textAlign: "left", paddingLeft: '10px'}}>
+            <h2>Request For Help</h2>
+            <Form
+                {...formItemLayout}
+                initialValues={props.initialValues}
+                onFinish={props.handleSubmit}>
+                <Form.Item label="Name" name="name"
```

Comment:
these components are available inside Volunteer form. I should have renamed it as Forms/ by now